controller/iteration_blackbox_test.go

```diff
@@ -537,12 +537,11 @@ func (rest *TestIterationREST) TestIterationStateTransitions() {
 	_, updated := test.UpdateIterationOK(rest.T(), svc.Context, svc, ctrl, itr1.ID.String(), &payload)
 	assert.Equal(rest.T(), startState, *updated.Data.Attributes.State)
 	// create another iteration in same space and then change State to start
-	userActive := false
 	itr2 := iteration.Iteration{
 		Name:       "Spring 123",
 		SpaceID:    itr1.SpaceID,
 		Path:       itr1.Path,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line. `UserActive` will automatically default to `false` when not set.

Reply:
this line is removed -> b92020c